jira-backport/jira-backport.py

```diff
@@ -0,0 +1,136 @@
+from jira import JIRA
```

Comment:
Any interest in defining this as a `uv` script and avoiding the need to install deps?

https://docs.astral.sh/uv/guides/scripts/#declaring-script-dependencies

Reply:
Absolutely, let me add it.

Reply:
Have you got go dependencies to bump?